lib/live-announcer.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { VisuallyHidden } from './visually-hidden';
+
+// see https://almerosteyn.com/2017/09/aria-live-regions-in-react
+
+const LiveAnnouncerContext = React.createContext();
+export const LiveAnnouncerConsumer = LiveAnnouncerContext.Consumer;
+export const useLiveAnnouncer = ({ message, live }) => {
+  const announce = React.useContext(LiveAnnouncerContext);
+  React.useEffect(() => {
+    if (announce) announce({ message, live });
+  }, [announce, message, live]);
+};
+
+const MessageBlock = ({ live, message = '' }) => (
+  <div role="log" aria-relevant="additions" aria-atomic="true" aria-live={live}>
```

Comment:
Why not include `text` in `aria-relevant`? (Making it the default value, `additions text`). As is, if there are 3 messages in quick succession, it's possible the first one won't have gone to nothing yet and so the third one will be interpreted as a text change, not an addition.

Reply:
I _think_ proving this out, when I generated a bunch of notifications (some with the same text), only the unique text ones or the first ones of a certain text got announced. I could be wrong about what's causing that, though. 
![image](https://user-images.githubusercontent.com/4480480/62962752-2ba5bd80-bdc5-11e9-84e2-d57ade2a3027.png)